.github/workflows/golang-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Run Golang Tests
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch.
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, reopened, edited, synchronised]
+    branches:
+      - master
+  # Also trigger when a new release has been created.
+  release:
+    types:
+      - created
+jobs:
+  install-and-run:
+    name: Install go and run tests
+    runs-on: ubuntu-latest
+    env:
+      CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.13.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.0'
```

Comment:
I'll assume this is to maintain equivalence with Travis.  Should we make this always use the latest version of Go?  If not now, then please leave a TODO.

Reply:
https://github.com/actions/setup-go/pull/32 implies that we can make this say '^1.' and it'll do the right thing.

Reply:
I've changed it to test using the latest version and added a TODO for future extension to other versions of Go.